### ISSUE TITLE
Front-end notes for items on non-PUL cards

### ISSUE THREAD

**Issue by jkotin** [Princeton-CDH/mep-django#693]
Scenario: I am investigating the circulation of (for example) _Blast_ in the lending library. I search for the periodical on the books page. I find one event. I click the cards tab on the Blast bibliography page and see the note, "No lending library cards available." It turns out the one event is for James Joyce, and the event is recorded on his cards at Buffalo.

My feature request: instead of "No lending library cards available," would it be possible to have a message saying that the item appears on the Joyce cards at Buffalo?

In other words: when clicking the cards tab from a bibliography page for an item on a non-PUL card, visitors would see an explanation about the location of the card.

As it stands, visitors can see such explanations when clicking on the cards tab from a member's page. For example:

https://shakespeareandco.princeton.edu/members/faulkner-norma/cards/
